templates/easyappointments/0/rancher-compose.yml

```diff
@@ -0,0 +1,75 @@
+version: '2'
+
+catalog:
+  name: 'Easy!Appointments'
+  version: 0.1.0
+  description: 'Open source appointment scheduler'
+  questions:
+    - variable: base_url
+      label: 'Base URL'
+      type: string
+      required: true
+    - variable: timezone
+      label: Timezone
+      type: string
+      default: UTC
+      required: true
+    - variable: version
+      label: Version
+      type: enum
+      default: 1.3.0
+      options:
+        - 1.3.0
+        - 1.3
+        - 1
+        - 1.2.0
+        - 1.2
+        - latest
+    - variable: mysql_host
+      label: 'Mysql Host'
+      description: 'Leave empty to spin up a mysql docker container'
+      type: string
+      required: false
```

Comment:
What do you think to replace `mysql_host` type string to type service??
```
    - variable: mysql_host
      label: 'Mysql Host'
      description: 'Leave empty to spin up a mysql docker container'
-      type: string
+      type: service
      required: false
```

Reply:
I'm not familiar with type service

Reply:
type `service` shows combo box wit a list of stack/services running at rancher.. could be exposed at service as link or external_link... Useful when you have more than one ip to expose and it's resilient to service update/upgrade...

Reply:
Um, yeah that's not how MySQL host is for. You only fill it out if you're doing an external connection